src/site/_plugins/markdown.js

```diff
@@ -36,7 +36,20 @@ const markdown = md({
     permalinkClass: 'w-headline-link',
     permalinkSymbol: '#',
     // @ts-ignore
-    slugify: (s) => slugify(s, {lower: true, remove: /[$*_+~.()'"!\-:@]+/g}),
+    slugify: (s) => {
+      let slug = slugify(s, {lower: true, remove: /[$*_+~.()'"!\-:@]+/g});
```

Comment:
@matthiasrohmer is this still needed now #8262 is merged?

If so, and you want to URL encode non-English chars, which I think [is technically the right thing](https://stackoverflow.com/questions/40636281/what-are-the-eligible-characters-in-a-urls-fragment-location-hash), but leads to ugly URLs when they seem to work fine without encoding them, then I think you need to change this as this code no longer works?

Reply:
Thanks for flagging here! I'll check and then update accordingly, but probably this is fixed with your change 🙂

Reply:
Good stuff. FYI I’ve a enhancement to that in #8549 as missed question marks in my first PR.